### DISCUSSION TITLE
set the default new window size?

### DISCUSSION THREAD

**Discussion by mattly** [zed-industries/zed#6676] [Feature Requests]
Whenever I open a new window in Zed, it's always maximized. This is never what I want. Is there a way to specify a default size?

**Comment by fuzenco** [comment]
My second most-wanted feature.

**Comment by pwfraley** [comment]
Since I use a tiling wm it would be nice if zed would not start maximized

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/community-links
